pkgs/by-name/kr/krep/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "krep";
+  version = "0.1.7";
+
+  src = fetchFromGitHub {
+    owner = "davidesantangelo";
+    repo = "krep";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-ffR5NVwvuwPdyEP2qKy7f+hX6tQIkMGyMXGopzQjbkI=";
+  };
+
+  buildPhase = ''
+    ENABLE_ARCH_DETECTION=0 make
+  '';
+
+  installPhase = ''
+    make install PREFIX=$out
```

Comment:
Probably you could add this to the makeFlags too, and omit the installPhase.

Reply:
Not sure how to do this. I added PREFIX to makeFlags but the `result` was empty.